src/update_uuids/modify_fileUuids_in_db.py

```diff
@@ -0,0 +1,49 @@
+import json
+import pymongo
+import requests as rq
+import sys
+import uuid
+
+
+def find_file_ids(project_uuid):
+    uuid_to_fileuuid = {}
+    azul_url = "https://service.azul.data.humancellatlas.org/index/files"
```

Comment:
Better to use an environment variable for the Azul URL. The hard coded value can be the default.